### PR TITLE
[excel] (Platform) Update for preview on family/home editions 

### PR DESCRIPTION
**Pull request by AlexJerabek**
Office Scripts for Excel is coming to consumer versions of M365. This PR updates the docs to include relevant storage information and consumer restrictions with PA.

### CODE REVIEW

File: docs/overview/script-storage.md [hunk 1]
```diff
@@ -38,20 +37,61 @@ To save a copy of a script to SharePoint, go to the **More options (…)** menu
...
+
 When you delete a script in Excel, it goes to your OneDrive or SharePoint recycle bin. To restore a deleted script, follow the steps listed in [How to recover missing, deleted or corrupted items in SharePoint and OneDrive for work or school](https://support.microsoft.com/office/3d748edf-c072-46c9-81a4-4989056ebc87). Restoring an **.osts** file returns it to the **All scripts** list.
 
 A deleted script is unshared with the workbook. When you restore a script, it does **not** retain its script access. You will need to share the script again.
 
 Restored scripts still work as expected with Power Automate flows. You don't need to recreate the flow connector.
 
+# [For personal/family](#tab/home)
+
+When you delete a script in Excel, it goes to your recycle bin. Restoring an **.osts** file returns it to the **All scripts** list.
```
Comment (micsull) [inline docs/overview/script-storage.md:77]: Unfortunately, we don't have scripts as actual osts files in consumer (yet).

They're just in the browser localstorage cache as entries there.  When you delete it... it's gone.  One reason why we're marked as "Preview" for now.

File: docs/overview/script-storage.md [hunk 2]
```diff
@@ -38,20 +37,61 @@ To save a copy of a script to SharePoint, go to the **More options (…)** menu
...
+# [For business/education](#tab/business)
+
+To give users who are outside of the SharePoint site access to the script, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
+
+> [!NOTE]
+> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access).
+
+# [For personal/family](#tab/home)
+
+Scripts can't be shared through workbooks with a home or family account. You can manually share the script files, but there are no in-client support options or change notifications.
```
Comment (micsull) [inline docs/overview/script-storage.md:61]: Similar to the comment below - the scripts aren't yet files.  Maybe rephrase the last sentence to:

"You can manually share the contents of scripts, but there are no in-client support options or change notifications."

File: docs/overview/script-storage.md [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
 
-> [!NOTE]
-> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access).
+# [For business/education](#tab/business)
 
-## OneDrive
+Office Scripts are stored in your OneDrive by default. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery. Excel only recognizes and runs a script if it's in your OneDrive folder, a Sharepoint folder, or shared with the workbook. This means Excel needs internet connectivity to access Office Scripts.
 
-The default behavior is that Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+### OneDrive
```
Comment (micsull) [inline docs/overview/script-storage.md:18]: OneDrive for Business (to help disambiguate the enterprise/consumer cases?)

Comment (AlexJerabek) [reply docs/overview/script-storage.md:18]: This header is only in the "business" tab. Did you view this on the Learn staging site?

Comment (micsull) [reply docs/overview/script-storage.md:18]: Ah, gotcha.  No, I'm looking at the raw markdown.

File: docs/overview/script-storage.md [hunk 4]
```diff
@@ -74,7 +77,7 @@ Restored scripts still work as expected with Power Automate flows. You don't nee
 
 # [For personal/family](#tab/home)
 
-When you delete a script in Excel, it goes to your recycle bin. Restoring an **.osts** file returns it to the **All scripts** list.
+Office Scripts are deleted permanently when when removed in Excel. This behavior will change before the feature leaves the preview phase.
```
Comment (micsull) [inline docs/overview/script-storage.md:80]: Minor nit - "when when"

**Review by micsull — CHANGES_REQUESTED** [review]

**Review by alison-mk — APPROVED** [review]
Approved with two minor comments. If you decide to incorporate the feedback, both of the comments would apply in multiple places (but I only left one comment per issue).
